podcasts/GeneralSettingsViewController.swift

```diff
@@ -351,23 +366,28 @@ class GeneralSettingsViewController: UIViewController, UITableViewDelegate, UITa
     }
 
     func tableView(_ tableView: UITableView, titleForFooterInSection section: Int) -> String? {
-        if section == 1 {
+        guard let sectionItems = tableData[safe: section] else { return nil }
+
+        switch sectionItems {
```

Comment:
Since its for footers, can we switch on `sectionItems.last` instead and then we wouldn't have to do the special treatment for `.intelligentPlaybackResumption` and we could remove the array from the cases?

Reply:
Changed in 893a499dfcdb104ad9f9c70acfb23d6e351729e9